src/platform/portduino/PortduinoGlue.cpp

```diff
@@ -137,6 +163,13 @@ void portduinoSetup()
         }
     }
 
+    uint8_t dmac[6];
+    getMacAddr(dmac);
+    if (dmac[0] == 128 && dmac[1] == 0 && dmac[2] == 0 && dmac[3] == 0 && dmac[4] == 0 && dmac[5] == 1) {
+        std::cout << "*** Blank MAC Address not allowed! " << std::endl;
```

Comment:
What about: `"MAC address should be set in 'config.yaml'"`? Then it's easier to figure out how to fix it.

Reply:
Yes. Still a WIP. I'll finalize the error message once the new options are available.